frontend/packages/dev-console/src/components/pipelines/detail-page-tabs/pipeline-details/PipelineVisualizationTask.tsx

```diff
@@ -98,18 +107,35 @@ const TaskComponent: React.FC<TaskProps> = ({
   status,
   name,
   isPipelineRun,
+  disableTooltip,
 }) => {
-  const taskData: K8sResourceKind = task.data;
-  const stepList = _.get(taskData, ['spec', 'steps'], []);
+  const stepList = _.get(task, ['data', 'spec', 'steps'], []);
   const stepStatusList: StepStatus[] = stepList.map((step) => createStepStatus(step, status));
   const showStatusState: boolean = isPipelineRun && !!status && !!status.reason;
   const visualName = name || _.get(task, ['metadata', 'name'], '');
   const path = pipelineRunName
     ? `${resourcePathFromModel(PipelineRunModel, pipelineRunName, namespace)}/logs/${name}`
     : undefined;
-  const visTask = (
-    <>
-      <div className="odc-pipeline-vis-task__connector" />
+
+  let content = (
```

Comment:
Same thing. Naming could be more elaborate.

Reply:
Called it `taskPill` as it's the rendering of the task pill item itself.